src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -571,6 +412,22 @@ public void TypeKindUpdate()
                 Diagnostic(RudeEditKind.TypeKindUpdate, "struct C", CSharpFeaturesResources.struct_));
         }
 
+        [Fact]
```

Comment:
This would go away if we implemented the parameter renaming, correct? Let's link an issue to this test case.